cmd/limactl/usernet.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/lima-vm/lima/pkg/networks/usernet"
+	"github.com/spf13/cobra"
+)
+
+func newUsernetCommand() *cobra.Command {
+	var hostagentCommand = &cobra.Command{
+		Use:    "usernet",
+		Short:  "run usernet",
+		Args:   cobra.ExactArgs(0),
+		RunE:   usernetAction,
+		Hidden: true,
+	}
+	hostagentCommand.Flags().StringP("pidfile", "p", "", "write pid to file")
+	hostagentCommand.Flags().StringP("endpoint", "e", "", "exposes usernet api(s) on this endpoint")
+	hostagentCommand.Flags().String("listen-qemu", "", "listen for qemu connections")
+	hostagentCommand.Flags().String("listen-fd", "", "listen for fd connections")
```

Comment:
```suggestion
	hostagentCommand.Flags().String("listen", "", "listen on a Unix socket and receive Bess-compatible FDs as SCM_RIGHTS messages")
```

Becase `listen-fd` sounds like to take an integer value that represents an FD

Reply:
Done